site/guide/9-configuration.md

```diff
@@ -17,26 +17,29 @@ is configured with. This means that no matter which configuration provider
-| `port`          | `u16`             | Port to serve on.                               | `8000`                  |
-| `workers`*      | `usize`           | Number of threads to use for executing futures. | cpu core count          |
-| `max_blocking`* | `usize`           | Limit on threads to start for blocking tasks.   | `512`                   |
-| `ident`         | `string`, `false` | If and how to identify via the `Server` header. | `"Rocket"`              |
-| `ip_header`     | `string`, `false` | IP header to inspect to get [client's real IP]. | `"X-Real-IP"`           |
-| `keep_alive`    | `u32`             | Keep-alive timeout seconds; disabled when `0`.  | `5`                     |
-| `log_level`     | [`LogLevel`]      | Max level to log. (off/normal/debug/critical)   | `normal`/`critical`     |
-| `cli_colors`    | `bool`            | Whether to use colors and emoji when logging.   | `true`                  |
-| `secret_key`    | [`SecretKey`]     | Secret key for signing and encrypting values.   | `None`                  |
-| `tls`           | [`TlsConfig`]     | TLS configuration, if any.                      | `None`                  |
-| `limits`        | [`Limits`]        | Streaming read size limits.                     | [`Limits::default()`]   |
-| `limits.$name`  | `&str`/`uint`     | Read limit for `$name`.                         | form = "32KiB"          |
-| `ctrlc`         | `bool`            | Whether `ctrl-c` initiates a server shutdown.   | `true`                  |
-| `shutdown`*     | [`Shutdown`]      | Graceful shutdown configuration.                | [`Shutdown::default()`] |
-
-<small>* Note: the `workers`, `max_blocking`, and `shutdown.force` configuration
-parameters are only read from the [default provider](#default-provider).</small>
+| key                  | kind              | description                                     | debug/release default   |
+|----------------------|-------------------|-------------------------------------------------|-------------------------|
+| `address`            | `IpAddr`          | IP address to serve on                          | `127.0.0.1`             |
+| `port`               | `u16`             | Port to serve on.                               | `8000`                  |
+| `workers`*           | `usize`           | Number of threads to use for executing futures. | cpu core count          |
+| `max_blocking`*      | `usize`           | Limit on threads to start for blocking tasks.   | `512`                   |
+| `ident`              | `string`, `false` | If and how to identify via the `Server` header. | `"Rocket"`              |
+| `ip_header`          | `string`, `false` | IP header to inspect to get [client's real IP]. | `"X-Real-IP"`           |
+| `proxy_proto_header` | `string`, `false` | Header identifying client to proxy protocol.    | `None`                  |
```

Comment:
Maybe this should link `ProxyProto` the way `LogLevel` does, for example?

Reply:
I linked `proxy_proto` the way `ip_header` does. I figured `ProxyProto` itself is not particularly useful here, as it is not used as the setting itself.